### ISSUE TITLE
Landing page template missing H1

### ISSUE THREAD

**Issue by maya** [uswds/uswds#2594]
Moving https://github.com/uswds/uswds-site/issues/451 to this repo on behalf of @nschonni.

@nschonni said:
>Looks like the other layouts have an H1 heading, but the landing layout was missed.
Maybe the hero callout should be used https://github.com/18F/web-design-standards-docs/blob/0564febab45eed87117808115a094dcbb213de99/_layouts/landing.html#L11

I said:
>Landing pages can be tricky bc in some cases the logo may be used as the `<h1>`, but it may be simpler to just make the hero title the `<h1>` in this case. Thanks again.

**Comment by maya** [comment]
Fixed in #2594. Will be release in USWDS 2.
